seahorn/jobs2/byte_buf_eq2/aws_byte_buf_eq_harness2.c

```diff
@@ -0,0 +1,55 @@
+/*
+ *
+ */
+
+#include <aws/common/byte_buf.h>
+#include <byte_buf_helper.h>
+#include <seahorn/seahorn.h>
+#include <utils.h>
+
+extern bool sea_is_modified(char *);
```

Comment:
these should eventually be moved to seahorn.h

Reply:
moved to seahorn.h ... will send another PR in seahorn project.